resources/Materials/TestSuite/locale/numericformat.mtlx

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<materialx version="1.38" cms="ocio" colorspace="lin_rec709">
+  <nodedef name="ND_simple_srf_surface" node="simple_srf">
```

Comment:
We've recently removed our legacy material examples referencing "simple_srf", in favor of renderable examples that reference real shading models such as Standard Surface and glTF PBR.  Perhaps we should update these new examples to use the latest approach as well?

Reply:
I'll update the tests to use a material node.